<dec f='qv2ray/test/catch.hpp' l='2572' type='void Catch::AssertionHandler::handleExceptionThrownAsExpected()'/>
<def f='qv2ray/test/catch.hpp' l='8257' ll='8259' type='void Catch::AssertionHandler::handleExceptionThrownAsExpected()'/>
